update.xml

```diff
@@ -736,6 +736,21 @@
             <forum>https://bit.ly/2P4aLmt</forum>
             <telegram>https://t.me/ghostrider_reborn</telegram>
         </lettuce>
+	    
+    <manufacturer id="Yu">
```

Comment:
this can be removed as manufacturer "YU" has already been defined above (line 727)

Reply:
ok

Reply:
corrected